tests/asdf_tests/test_asdf_core.py

```diff
@@ -1,18 +1,32 @@
 """Tests asdf implementations of core module."""
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
 import numpy as np
 import pandas as pd
 import pytest
 import xarray as xr
 from asdf import ValidationError
+from fs.memoryfs import MemoryFS
+from fs.osfs import OSFS
 from scipy.spatial.transform import Rotation
 
 import weldx.transformations as tf
+from weldx.asdf.tags.weldx.core.file import ExternalFile
 from weldx.asdf.utils import _write_buffer, _write_read_buffer
 from weldx.constants import WELDX_QUANTITY as Q_
 from weldx.core import MathematicalExpression as ME  # nopep8
 from weldx.core import TimeSeries
 from weldx.transformations import WXRotation
 
+
+def get_test_name(param):
+    """Get the test name from the parameter list of a parametrized test."""
+    if isinstance(param, str) and param[0] == "#":
+        return param[1:]
+    return ""
```

Comment:
This is a duplication of a function in the test file of the core module. It`s needed for the exception. It produces a clearer output of the test results. Will move it to `_helpers.py`